src/main/java/application/AbstractBaseController.java

```diff
@@ -87,6 +88,9 @@
     protected FormplayerRaven raven;
 
     @Autowired
+    protected FormSendCalloutHandler formSendCalloutHandler;
```

Comment:
I think I'm missing why it doesn't make the most sense for the FormSession to not instantiate this directly. It seems like there's a ton of bootstrapping that's being done to pass this down the chain

Reply:
Autowiring is more idiomatic Spring and much more feature-ful than direct instantiation (giving us access to the entire set of autowired properties/objects). Ideally we'd autowire the FormSession object as well and then we could skip this long handoff chain - good future refactor